js/src/lib/components/PipelineTag/PipelineTag.svelte

```diff
@@ -1,14 +1,18 @@
 <script lang="ts">
-	import { PIPELINE_DATA } from "../../interfaces/Types";
-	import PipelineIcon from "../PipelineIcon/PipelineIcon.svelte";
+import type { PipelineType } from '../../interfaces/Types';
 
-	export let classNames = "";
-	export let pipeline = "";
+import { PIPELINE_DATA } from '../../interfaces/Types';
+import PipelineIcon from '../PipelineIcon/PipelineIcon.svelte';
+
+export let classNames = '';
+export let pipeline = '';
+
+$: pipelineName = pipeline ? PIPELINE_DATA[<PipelineType>pipeline]?.name ?? pipeline : pipeline;
```

Comment:
formatting needs to be reversed here too

Reply:
Done !